File: BE/src/main/java/com/example/LAGO/domain/HistoryChallengeNews.java
package com.example.LAGO.domain;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.*;

import java.time.LocalDateTime;

@Entity
@Table(name = "\"HISTORY_CHALLENGE_NEWS\"")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class HistoryChallengeNews {

    @Id
    @Column(name = "challenge_news_id")
    private Integer challengeNewsId;

    @Column(name = "challenge_id")
    private Integer challengeId;

    @Column(name = "title")
    private String title;

    @Column(name = "content")
    private String content;

    @Column(name = "published_at")
    private LocalDateTime publishedAt;
}

File: android/LAGO/app/src/main/java/com/lago/app/domain/entity/HistoryChallengeNews.kt
package com.lago.app.domain.entity

data class HistoryChallengeNews(
    val challengeNewsId: Int,
    val challengeId: Int,
    val title: String,
    val content: String,
    val publishedAt: String,
    val imageUrl: String = "" // 이미지 URL 필드 추가
)